sentry_streams/tests/adapters/arroyo/test_adapter.py

```diff
@@ -51,7 +53,12 @@ def test_kafka_sources() -> None:
     assert sources.get_consumer("source1") is not None
 
 
-def test_adapter(broker: LocalBroker[KafkaPayload], pipeline: Pipeline) -> None:
+def test_adapter(
+    broker: LocalBroker[KafkaPayload],
+    pipeline: Pipeline,
+    metric: IngestMetric,
```

Comment:
I'll probably stop spamming the `IngestMetric` type everywhere in tests and examples, and use a different one instead